content/rs/technology-behind-redis-enterprise.md

```diff
@@ -1,102 +1,73 @@
 description:
 weight: 10
 alwaysopen: false
 categories: ["RS"]
 ---
-RS's unique and patented technology was developed to meet these main
-objectives:
-
-- Decouple data path from cluster management
-- Ensure consistent top performance
-- Simplify and automate operations
-
-## Decouple data path from cluster management
-
-RS's architecture completely decouples the data path from the cluster
-management in order to optimize the operations of each of these
-components.
-
-### Data path
-
-The data path is based on multiple zero-latency, multi-threaded proxies
-that reside on each of the cluster nodes to mask the underlying complexity
-of the system. This model supports multiple proxies per Redis
-database and permits the use of any regular or cluster-aware Redis
-client. This allows the user's code base to work as-is. Each proxy
-forwards client requests to the relevant Redis servers (shards). When
-new shards are added to a database, the proxy handles the new routing
-rules for the application's requests transparently, by immediately
-scaling the database performance and memory capacity without any changes
-to the application itself.
-
-The proxies also support Memcached's text and binary protocols and
-perform in-flight translation between Memcached and Redis protocol
-semantics. This allows Memcached users to enjoy many of the features
-that are not available with open source Memcached, such as built-in
-replication, auto-failover, data persistence backups and scaling
-(out/in) without losing data.
+
+Redis Enterprise was built to simplify operations while providing seamless scalability, top performance, and high availability.
+
+Here are some of the features and design decisions that make this possible.
+
+## Data access and cluster management
+
+A Redis Enterprise Cluster separates data access from cluster
+management. This decoupled architecture results in increased reliability,
+top performance, and simplified operations.
+
+### Data access
+
+A Redis Enterprise Cluster hosts one or more Redis databases. Access to these databases runs through a multi-threaded proxy that lives on each cluster node. These proxies efficiently and transparently route queries to the appropriate underlying Redis instance.
+
+You can use any standard Redis client to access a Redis Enterprise database via the proxy. The proxies also support cluster-aware clients for applications requiring the lowest possible latency.
+
+In addition, the proxies understand Memcached's text and binary protocols. Applications using Memcached can immediately start using Redis Enterprise and take advantage of its many features, including built-in replication, auto-failover, data persistence, backups, and linear scaling.
 
 ### Cluster management
 
-The cluster manager is a sophisticated governing function that provides
-capabilities such as resharding, rebalancing, auto-failover,
+This cluster manager consists of a number of software components that monitor
```

Comment:
"This cluster manager" where was it defined before? Looks to me like it should be "The cluster manager"